src/pages/App.tsx

```diff
@@ -86,8 +87,10 @@ export default function App() {
             <Polling />
             <TopLevelModals />
             <Switch>
-              <Route exact strict path="/vote" component={Vote} />
-              <Route exact strict path="/vote/:governorIndex/:id" component={VotePage} />
+              <Suspense fallback={<Loader />}>
+                <Route exact strict path="/vote" component={Vote} />
```

Comment:
I think `exact` prevents this from matching for sub-paths of `/vote`, and needs to be omitted.